_posts/2022/2022-01-26-dev-experience-2022.md

```diff
@@ -0,0 +1,89 @@
+
+I used to spend a lot of time on academics (your typical *"ghissu"*). But I tried to attend most of the intro talks held by STC and got to know about SDSLabs. I knew about the group from the freshman guide too and had checked out some of their apps and projects.
+
+A couple of weeks later, SDSLabs announced the __Syntax Error__ hackathon. It was the first big event that happened in my first year (and was my *first* hackathon too!) so I was pretty psyched about it.
+
+I ended up creating a 2D space shooter for my submission. Though I didn't make it in the top 10, It was still a great experience for me and is a fond memory (*My first hackathon*, first of many)
+
+Around this time I started developing an interest in Information Security and CTFs too. I attended some workshops by InfoSecIITR and found it very cool.
+
+Fast Forward to...
+
+#### The Recruitments
+
+SDSLabs held many events for recruitments (Syntax Error, Makers, noobCTF) and a recruitment test. I attempted their recruitment test which was mostly problem solving and basic programming questions. There was a section on technological advancements and questions like *"Why do you think coding is a superpower?"*. I found this very interesting as this was not there in other groups' test and though this section carried no weightage, I attempted it (As it interested me).
+
+A few days later, I got a call for interview. The interview was conducted in 2 parts. First one was basic problem solving, and we also discussed internet protocols and basic web vulnerabilities.
+
+The 2nd round was based on my interests, how I would tackle a real world problem and any questions I would like to ask SDSLabs. I was asked to create the Tic-Tac-Toe game and a library management system. This was also a fanboy moment for me as Twarit Waikar was one of the interviewers (SDSLabs 2021 alumnus). He is a game developer and worked extensively on [Rubeus](https://github.com/sdslabs/Rubeus) and [Rootex](https://github.com/sdslabs/Rootex) game engines. I had read his blogs and looked at his games and the game engines and was pretty impressed by that (*Making a game engine seemed a very huge undertaking to me*). I asked them about game engines and game development and web security.
+
+## The Second Semester
+
+#### The initiation
+
+I got a phone call informing me about my induction to SDSLabs. We had an intro chapo (online, coz covid) and got to know everyone there. We were added on the workspace apps and this got the cart rolling.
+
+The semester was filled with lectures on various subjects (Git, Bash scripting, Web development, Game development) and we has follow up assignments. The follow up assignments spurred me to learn about something new and implement it within a deadline.
+
+After this, we were explained about all the apps and projects going on in SDSLabs. We were briefed about their status, what technology they are built upon, what problems they solve and what more work needs to be done.
```

Comment:
After the assignment season was concluded, we were introduced to the active apps and projects in SDSLabs. We were briefed on the status for each project and a received an overview of the tech stack, ongoing work, future issues and overall vision.